src/cjs_parse.rs

```diff
@@ -131,6 +132,44 @@ impl CjsVisitor {
       _ => {}
     }
   }
+
+  fn visit_exports_right_expr(&mut self, right_expr: &Expr) {
+    match right_expr {
+      Expr::Object(object_lit) => {
+        for prop in &object_lit.props {
+          match prop {
+            PropOrSpread::Prop(prop) => {
+              if let Some(prop_name) = get_prop_name(prop) {
+                if is_supported_object_prop(prop) {
+                  self.add_export(prop_name);
+                } else {
+                  self.add_unsafe_getter(prop_name);
+                }
+              }
+            }
+            PropOrSpread::Spread(spread) => {
+              if let Some(require_value) = get_expr_require_value(&spread.expr)
+              {
+                self.add_reexport(require_value);
+              }
+            }
+          }
+        }
+      }
+      Expr::Call(call_expr) => {
+        // module.exports = require(...);
+        if let Some(require_value) = get_call_expr_require_value(call_expr) {
+          self.set_reexport_assignment_value(require_value);
+        }
+      }
+      Expr::Assign(right_assign_expr) => {
+        self.in_module_exports = true;
+        self.visit_assign_expr(right_assign_expr);
+        self.in_module_exports = false;
```

Comment:
Seems like we can remove this extra state and just do?

```suggestion
        self.visit_exports_right_expr(&right_assign_expr.right);
```

That seems to pass all the current tests, but is there a scenario where `self.in_module_exports` is necessary? It would be best to not introduce more state if possible.

Reply:
Thanks for your suggestion, indeed `in_module_exports` is redundant ~